specification/repository.md

```diff
@@ -107,6 +107,7 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
   - SHOULD have badges on the `README.md` for other relevant things including artifacts
   - MUST have getting started information in `README.md`
   - MUST have troubleshooting information in `README.md`
+  - MUST link to official Splunk docs if they exist
```

Comment:
other bullets repeat `README.md` all the time. Do you think it should be removed? I would prefer to keep consistency.

Reply:
Fixed!